scripts/install-packages.sh

```diff
@@ -4,11 +4,8 @@
 # packages to be installed
 ################################################################################
 
-PKGS_TO_INSTALL=(archlinux-keyring base bash bind btrfs-progs cronie curl dash dosfstools e2fsprogs exfatprogs findutils gcc git git-lfs htop iotop iperf iperf3 iputils less linux-firmware lm_sensors lsb-release lsof man man-db man-pages mlocate nano networkmanager opendoas openssh openssl pacman-contrib sudo tar tmux vim wget which xfsprogs xz zip zsh zstd)
-
-# extra pkgs
-#PKGS_TO_INSTALL+=(aria2 base-devel bat btop choose dog dua-cli dust exa fd hyperfine namcap neovim nload procs ripgrep rsync rustup skim tealdeer tre tree unrar unzip wget yt-dlp zsh-autosuggestions zsh-completions zsh-syntax-highlighting)
-
+PKGS_TO_INSTALL=(base networkmanager openssh vim)
```

Comment:
I see you only kept 4 packages `base networkmanager openssh vim`. I suggest you add at least a few more packages that may look "additional" but are important to diagnose why a fresh new image is not working. Packages like `iputils`, `bind`, `nano` are necessary, not for a functioning system, but to diagnose _why_ it isn't working.

Reply:
More conversation about packages below. Resolving here.